source/_integrations/konnected.markdown

```diff
@@ -93,6 +93,16 @@ The settings UI starts by having you configure the general behavior of each zone
 
 **Configure additional states for this zone:** Selecting "No" will complete configuration for the zone and proceed to options for the next zone. Select "Yes" if you need to create additional output states for this zone.  
 
+#### Using Settings UI to Configure Additional Panel Behavior
+
+Once all zones are configured you'll be presented with configuration for additional panel behaviors.
+
+**Blink panel LED on when sending state change:** The desired LED behavior for the panel
+
+**Override default Home Assistant API host panel URL:** The Konnected Alarm Panel post sensor states back to the Home Assistant API.  If this value is unchecked the panel will default postbacks using `base_url` in the `http` component. If you've set `base_url` to an external hostname, then you'll want to check this field and set the **Override API host url** to your _local_ IP address and port (e.g., `http://192.168.1.101:8123`).
+
+**Override API host URL (optional):** The host info to use if you checked **Override default Home Assistant API host panel URL** in the step above.  This is ignored if **Override default Home Assistant API host panel URL** is unchecked.
```

Comment:
```suggestion
**Override API host URL (optional):** The host info to use if you checked **Override default Home Assistant API host panel URL** in the step above. This is ignored if **Override default Home Assistant API host panel URL** is unchecked.
```

Reply:
Sorry - old habit throwing that extra space in there 🤕. I know it's against guidelines...

Reply:
Hehe don't worry, we'll catch them 😉